lib/main_common.pm

```diff
@@ -458,7 +458,9 @@ sub load_extra_tests() {
         # dependency of git test
         loadtest "console/sshd";
         loadtest "console/zypper_ref";
-        loadtest "console/zypper_info";
+        if (sle_version_at_least('12-SP2') || check_env('DISTRI', 'opensuse')) {
```

Comment:
> "checked feature is implemented from SLE12SP2" in the commit message sounds weird. Probably you meant "implemented in SLE 12 SP3"?

if is this test has to test feautre implemented from SLE12SP3 .. is wrong , because tests something else